noether_tpp/test/tool_path_modifier_utest.cpp

```diff
@@ -147,8 +192,10 @@ TEST_P(OneTimeToolPathModifierTestFixture, TestOperation)
   auto modifier = GetParam();
 
   // Create several arbitrary tool paths
-  std::vector<ToolPaths> inputs = { createArbitraryToolPath(4, 1, 10, createIdentityWaypoint),
-                                    createArbitraryToolPath(4, 1, 10, createRandomWaypoint) };
+  std::vector<ToolPaths> inputs = { createArbitraryToolPath(4, 2, 10, createIdentityWaypoint),
+                                    createArbitraryToolPath(4, 2, 10, createRandomWaypoint),
```

Comment:
While I agree that "createRandomWaypoint" is a good way to test code during development, I don't think it is a good idea to implement a unit test.  We want to make sure that any failure in the unit test is reproduceable, and that all the planners are passing the same tests.  I would be fine using the function to generate and save off test cases.

I am, of course, open to hearing your thoughts behind this.

Reply:
The idea of having "random" waypoints is to test the functionality of the waypoint orientation modifiers on something that is not trivial and doesn't happen to always be correct. I'll look into adding a random seed to make the random sequence always the same